src/ex4/client/src/redux/actions/itemsViewActions.js

```diff
@@ -0,0 +1,21 @@
+import actionTypes from "../actions/constants/index";
+
```

Comment:
try not to name actions with "dispatch" in its name because it's confusing.
actions should be named like any regular function and keep the dispatch only for the `dispatch` function of redux that they trigger inside